lib/views/navigation/bottom_navigation_bar/navigation_bar_routes.dart

```diff
@@ -0,0 +1,58 @@
+import "package:flutter/material.dart";
+import "package:proxima/utils/ui/not_implemented.dart";
+import "package:proxima/views/content/feed/post_feed.dart";
+import "package:proxima/views/navigation/routes.dart";
+
+/// This enum is used to create the navigation bar routes.
+/// It contains the name and icon of the routes.
+enum NavigationbarRoutes {
+  //TODO set the routes for the pages
```

Comment:
I don't understand this TODO. Isn't it already done?

Reply:
Good catch !